README.md

```diff
@@ -29,54 +29,82 @@ tool in [Don't Panic: Kubernetes and Docker](https://blog.k8s.io/2020/12/02/dont
 and on the Mirantis
 [blog](https://www.mirantis.com/blog/mirantis-to-take-over-support-of-kubernetes-dockershim-2/).
 
-## Build and install
+## Using cri-dockerd
 
-To begin following the build process for this code, clone this repository in your local environment:
+### Install
+
+The easiest way to install `cri-dockerd` is to use one of the pre-built binaries or
+packages from the [releases page](https://github.com/Mirantis/cri-dockerd/releases).
+There are numerous supported platforms and using a pre-built package will install
+the binary and setup your system to run it as a service.
+
+### Installing manually
+
+If you would like to install the project manually, you will need to place the binary
+somewhere in your `PATH` and setup a service to run it. The following commands is
+a manual install for a Linux system using systemd:
 
 ```shell
-git clone https://github.com/Mirantis/cri-dockerd.git
+install -o root -g root -m 0755 cri-dockerd /usr/local/bin/cri-dockerd
+install packaging/systemd/* /etc/systemd/system
+sed -i -e 's,/usr/bin/cri-dockerd,/usr/local/bin/cri-dockerd,' /etc/systemd/system/cri-docker.service
+systemctl daemon-reload
+systemctl enable cri-docker.service
```

Comment:
This was part of the original install docs so please correct my understanding if I'm missing something

The socket is a part of the service. So wouldn't this actually be the other way around and the socket doesn't need to be enabled since the service is managing it?

Reply:
Ah, I must have missed it there. A socket unit activates a service unit; systemd listens on the socket, and when something connects, it starts the service and hands off the socket via a FD.

You're thinking of the traditional model where a service starts and listens on a socket; indeed, if you're not using socket activation, there's no need to activate the socket unit. My proposed change will delay the start of cri-dockerd until something tries to connect, which I think is a pretty sane default.

Reply:
If you want to encourage people to eagerly start cri-dockerd (with the `.service`) instead of autostarting it on demand (with the `.socket`), you should add `--now` -- as it is, the service won't run until reboot or an explicit start.

Reply:
I just realized I removed the wrong line from the discussion. I meant to leave the line to enable the socket with `--now`

Reply:
You don't have to enable the service at all, only the socket. But people do it anyway, either because they want to cri-dockerd to start up no matter if it is used or not - or maybe because they just copy paste stuff from containerd....